functests/performance/performance.go

```diff
@@ -103,6 +113,20 @@ var _ = Describe("performance", func() {
 		})
 	})
 
+	Context("Additional kernel arguments added from perfomance profile", func() {
+		It("Should set additional kernel arguments on the machine", func() {
+			if profile.Spec.AdditionalKernelArgs != nil {
+				for _, node := range workerRTNodes {
+					cmdline, err := nodes.ExecCommandOnMachineConfigDaemon(testclient.Client, &node, []string{"cat", "/proc/cmdline"})
+					Expect(err).ToNot(HaveOccurred())
+					for _, arg := range profile.Spec.AdditionalKernelArgs {
+						Expect(cmdline).To(ContainSubstring(arg))
+					}
+				}
+			}
+		})
+	})
```

Comment:
Nice, almost there. I'd add an addional kernel argument, something innocent but easy to find like "quiet", "norhgb" or anything else fit to the task you can find, to prove it is correctly passed down. WDYT?

Reply:
I think you can add anything (eg. cnf.test=1), kernel should ignore unknown values

Reply:
It might also be a good idea to add two: cnf.testB=1 and cnf.testA=2 and test it was not reordered (because we saw that with hugepages).

Reply:
I added the reasonable values under here https://github.com/openshift-kni/performance-addon-operators/pull/133/files#diff-94b0c90ad9b93901aa923f6058f1336c  for testing CI.

Reply:
@MarSik  @fromanirh   how about now?  checking all the arguments as one subset that should appear in proc/cmdline ?  if the order is not the same it will fail.

Reply:
Isn't it just testing that all are there (ContainSubstring)?

Reply:
Substring is "arg1 arg2 arg3" .   it will fail if it wont be the exact substring, for example if in proc/cmdline we have "arg2 arg1 arg3" then the test fails.

Reply:
Oh, ok, old page. But maybe move the additionalArgs definition out of the loop